content/en/developers/authorization/oauth2_in_datadog.md

```diff
@@ -28,23 +28,25 @@ This page provides a step-by-step overview on how to implement the OAuth protoco
 
    To learn how to derive the `code_challenge` parameter, see the [PKCE](#authorization-code-grant-flow-with-pkce) section. Your application must save `code_verifier` for the token request in Step 5.
 
-   - In order to build the URL for this post request, use the `site` query parameter that is provided on the redirect to your `onboarding_url`. 
+   - To build the URL for this GET request, use the `domain` query parameter that is provided on the redirect to your `onboarding_url`. 
    - This parameter is only provided if the user initiates authorization from the Datadog integration tile. See the [Initiate authorization from a third-party location](#Initiate-authorization-from-a-third-party-location) section for more options if the user chooses to initiate authorization externally.  
-   - For example, `site` may be `https://app.datadoghq.com`, `https://app.datadoghq.eu`, `https://us5.datadoghq.com`, or it might have a custom subdomain that must be accounted for, such as `https://<custom_subdomain>.datadoghq.com`. 
-   - To handle multiple Datadog sites dynamically, you can append the Datadog `authorize` path to this constructed URL.
+   - The `domain` query parameter provides the [Datadog site][17] that the authorizing user is in, and is required to construct the URL for this GET request to the Authorize endpoint: `https://api.<domain>/oauth2/v1/authorize?...`.
+
+<div class="alert alert-info">The `domain` parameter was introduced in July 2023, and previous OAuth clients use the `site` parameter. While the `site` parameter is supported, Datadog recommends using the `domain` parameter. </div>
```

Comment:
```suggestion
<div class="alert alert-info">The `domain` parameter was introduced in July 2023, and previous OAuth clients use the `site` parameter. While the `site` parameter is supported, Datadog recommends using the `domain` parameter. </div>
```
Not sure if we want to call out that it's partially supported since it won't work for endpoints like `logs`

Reply:
I don't think we need to call this out specifically